openslides_backend/action/actions/group/create.py

```diff
@@ -13,5 +13,9 @@ class GroupCreate(CreateAction):
     model = Group()
     schema = DefaultSchema(Group()).get_create_schema(
         required_properties=["name", "meeting_id"],
-        optional_properties=["permissions"],
+        optional_properties=[
+            "permissions",
+            "default_group_for_meeting_id",
+            "admin_group_for_meeting_id",
```

Comment:
Please remove this: this is not part of the action payload, as described in https://github.com/OpenSlides/OpenSlides/wiki/group.create. If you need this for internal reasons, you would have to create a separate action for this or (probably more complex) implement the possibility to mark some payload elements as internal. Also, on another note, this seems like it belongs to #486 and not this PR.

Reply:
Except, @FinnStutzenstein: Does the user have the option to change these default groups or are they fix once the meeting is created? If the former, we could talk about adding this to the payload.

Reply:
They are fixed and only touched on meeting.create

Reply:
Removed. These fields are part of meeting.create